### ISSUE TITLE
The charts cannot be seen....not in the vercel app deployment as well

### ISSUE THREAD

**Issue by UtsavKadecha10** [marcinsuski/finance-dashboard#1]

**Comment by marcinsuski** [comment]
This was a test project with a separate server. If you cannot see the charts on the vercel deployment, it means server is restarting - simply wait around 5-10 seconds or refresh the page after this time and it shows all the charts. It's not a code issue but the server that is simply a free tier and shuts down if the website is not visited often. I do not intend to change this as it is only a demo. 
If, on the other hand, you have an issue with the charts on your local dev server, please tell me how can I recreate the issue.

**Comment by UtsavKadecha10** [comment]
Yes I do have an issue on the local dev server, it did show the charts on
hosted website, but the issue on the local dev still persists.
